### DISCUSSION TITLE
/etc/input.d/nimf.conf은 어떤 application에서 사용되는 건가요?

### DISCUSSION THREAD

**Discussion by june3474** [hamonikr/nimf#86] [Q&A] [answered]
제목 그대로 입니다. 
nimf를 설치(소스 컴파일 설치)하면 /etc/input.d/nimf.conf 파일이 설치되는데 이 파일이 어떤 프로그램에서 사용되는 건가요?

/etc/X11/xinit/xinput.d/nimf.conf는 im-config나 im-chooser와 같이 imsettings를 사용하는 App에서 사용된다는 건 구글 검색결과 알게 되었는데 /etc/input.d/는 정보가 없네요

**Comment by chaeya** [comment, marked as answer]
이 파일은 https://github.com/hamonikr/nimf/blob/master/bin/nimf-settings/nimf-settings.c 파일에서 사용됩니다.

이 부분은 사용자가 nimf-settings GUI를 통해 입력기 설정을 변경할 때, 사용자 설정 디렉토리 내의 environment.d 폴더에 nimf.conf 파일의 경로를 생성하고, /etc/input.d/nimf.conf에 대한 심볼릭 링크를 생성해서, 시스템 전역 설정과 사용자별 설정 간의 일관성을 유지하는 부분입니다.

**Comment by june3474** [comment]
감사합니다. ^^
